packages/core/admin/admin/src/content-manager/components/BlocksEditor/plugins/withStrapiSchema.ts

```diff
@@ -1,11 +1,13 @@
-import { Element, Transforms } from 'slate';
+import { type Text, Node, Editor, Element, Transforms } from 'slate';
+
+const isText = (node: unknown): node is Text => {
+  return Node.isNode(node) && !Editor.isEditor(node) && node.type === 'text';
+};
```

Comment:
I wish I could export and reuse this type, as it's needed in 3 separate files. But when I export it, I get a TS build error:

>Exported variable 'isText' has or is using name 'TextInlineNode' from external module "strapi/packages/core/types/dist/types/core/attributes/blocks" but cannot be named.

that I haven't been able to fix.

Reply:
I just realised Slate already provides Text.isText() method https://docs.slatejs.org/api/nodes/text#text.istext-value-any-greater-than-value-is-text, where its checking if text property is present or not, which should be enough I guess.

Reply:
It can be useful indeed, can you just check if it works for us considering we have `type: text` on our leaves?